content/10.diagnostics.md

```diff
@@ -214,24 +214,37 @@ This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG
 It reported that IgG could be detected within two days of the onset of fever but that IgM could not be detected any earlier, a pattern they compared to findings in another disease caused by a HCoV, Middle East respiratory syndrome (MERS).
 Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
 
-Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036].
-One study assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
+Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].
```

Comment:
Generally speaking, the consensus is that yes infection does confer sustained immunity but that after the 6-8 month period the immunity does wane and there is increased risk of reinfection for some. It is not clear though yet, whether there is still protection from reinfection beyond that time point, and we really don't know what the story will be with the t-cell and b-cells pool. My guess is there must be some protection, but how protective that is to prevent reinfection, I really do not know. The group that produced the Gaebler study have kept some patients to do a 1 and 2 year post infection study, which will certainly provide answers to that. 

I hear your concerns though, inferring protection from these types of studies is not always indicative of reality, but my perception is that examining antibody responses and assessing neutralizing activity seems to be the gold standard for these folks.  

PS I highly recommend you listen to Christian Gaebler who is on an episode of the podcast TWIV, which I also recommend. He gives an overview of his study and what is to come for sustained immunity research with SARS-CoV-2

Reply:
Thank you so much for explaining @RLordan! And thank you for the rec, I just subscribed to that podcast!

That definitely makes sense! Maybe something we could mention in the conclusions for this paper in terms of future directions? That's fantastic they're geared up for a longitudinal study!